### ISSUE TITLE
[Accessibility issue-iOS] Contradictory message displayed when click on Subscribe button for xhr.

### ISSUE THREAD

**Issue by harshamulpuri2913** [ampproject/amphtml#3881]
## Short description of your issue: within iOS device (IPAD / IPOD), a contradictory message is displayed as "Everything looks good, you can submit now!

Oops!, We apologies something went wrong. Please try again later"

Test URL: https://amp-mk-1.herokuapp.com/examples.build/forms.amp.max.html

Pre-requisite: Open IPAD--> Settings --> General --> Accessibility --> Voiceover , Set to ON
## How do we reproduce the issue?
1. Logon to IPAD and open the safari browser
2. Open test URL provided above
3. Under "Subscribe to our weekly newsletter (xhr)", enter your name input as "XYZ" & your email input as "x@gmail.com"
4. click on Subscribe button

Actual results: 
1. Reads out as "Subscribe dimmed"

Expected result: 
Based on successful or unsuccessful submissions below messages should be displayed and read out appropriately:
1. Should read out as "Everything looks fine, you can submit now!"
OR
2. Oops ! We apologise something went wrong. Please try again later. "
## What browsers are affected? Safari , Chrome

All browsers? Some specific browser? What device type?
Device details: IPAD 
iOS version: 9.3.2 (13F69)

Attachments: 
![img_0043](https://cloud.githubusercontent.com/assets/20247457/16532961/596445e6-3f8a-11e6-9dc0-c6f3223945bb.PNG)

**Comment by cvializ** [comment]
I just tested this and I think in the meantime is has been resolved through other changes. If you are still experiencing this error feel free to reopen!
